src/frontend/src/styles/pages/Dashboard.scss

```
@@ -30,28 +39,16 @@
 
 .dashboard-page__logo {
     max-width: 100%;
-    object-fit: cover;
-    width: 500px;
-}
-
-// .dashboard-page__heading {
-//     @include type-style('productive-heading-05');
-// }
-
-.dashboard-page__subheading {
-    // @include type-style('productive-heading-03');
-    font-weight: 600;
+    height: auto;
```

we only need to set the width to 100% with the simplified structure (ie. remove max-width and auto height). Good job simplifying this!

```suggestion
    width: 100%
```